packages/@ember/-internals/routing/lib/system/router.ts

```diff
@@ -517,7 +517,7 @@ class EmberRouter extends EmberObject {
     @public
   */
   transitionTo(...args: unknown[]) {
-    if (resemblesURL(args[0] as string)) {
+    if (resemblesURL(args[0])) {
```

Comment:
Mind making an issue for a random aside that this made me think about?

If you call `.transitionTo` with a thing that "resembles a URL" but also pass more than one string arg, we silently swallow the options 😭

Reply:
I figured something like that might happen given the way this works. 😬 Opening the issue now!

Reply:
[issue!](https://github.com/emberjs/ember.js/issues/18760)